### ISSUE TITLE
SPIRV-Cross -> patch could not be applied  (MPV borks)

### ISSUE THREAD

**Issue by pat357** [jb-alvarado/media-autobuild_suite#1255]
When building SPIRV-Cross (32 and 64bit), the following is printed on screen:

``` bash
+ [[ git = svn ]]
+ [[ git = hg ]]
+ [[ git = git ]]
+ [[ -n https://github.com/KhronosGroup/SPIRV-Cross.git ]]
+ git remote set-url origin https://github.com/KhronosGroup/SPIRV-Cross.git
+ git config remote.origin.fetch '+refs/heads/*:refs/remotes/origin/*'
+ [[ -f .git/refs/heads/ab-suite ]]
+ git checkout ab-suite
Already on 'ab-suite'
++ vcs_getlatesttag origin/HEAD
++ local ref=origin/HEAD
++ [[ -n git ]]
++ [[ git != git ]]
++ local tag
++ [[ origin/HEAD = \L\A\T\E\S\T ]]
++ [[ origin/HEAD = \G\R\E\A\T\E\S\T ]]
++ [[ origin/HEAD != \o\r\i\g\i\n\/\H\E\A\D ]]
++ echo origin/HEAD
+ git reset --hard origin/HEAD
HEAD is now at 4104e363 Merge pull request #1027 from KhronosGroup/fix-1026
++ git rev-parse HEAD
+ oldHead=4104e363005a079acc215f0920743a8affb31278
+ set +x
18:06:48 + SPIRV-Cross git .......................... [Recently updated]
master...taisei-project:meson.diff
        Patch couldn't be applied with 'patch'. Continuing without patching.
18:06:49 Running meson in SPIRV-Cross-git
```

This wouldn't be so bad  as the building continues without any errors, but the final MPV with this lib linked borks out on the following:
```
mpv --no-config --vo=gpu --profile=gpu-hq --hwdec=d3d11va <mediafilename>
```
A white screen is shown an the player exits, no video is shown.
The mpv-log shows:
```
[   0.523][d][vo/gpu/d3d11] [ 71] 
[   0.523][d][vo/gpu/d3d11] [ 72] SPIRV_Cross_Output main(SPIRV_Cross_Input stage_input)
[   0.523][d][vo/gpu/d3d11] [ 73] {
[   0.523][d][vo/gpu/d3d11] [ 74]     _98 = stage_input._98;
[   0.523][d][vo/gpu/d3d11] [ 75]     frag_main();
[   0.523][d][vo/gpu/d3d11] [ 76]     SPIRV_Cross_Output stage_output;
[   0.523][d][vo/gpu/d3d11] [ 77]     stage_output._315 = _315;
[   0.523][d][vo/gpu/d3d11] [ 78]     return stage_output;
[   0.523][d][vo/gpu/d3d11] [ 79] }
[   0.523][e][vo/gpu/d3d11] Failed to create Texture2D: De parameter is onjuist. (0x80070057)
``` 
The "official" build downloaded from MPV.IO build today (16/6/2019) does NOT have this issue, so I hope being able to apply the patch will resolve this for the build created by the MABS.

**Comment by wiiaboo** [comment]
There's no official builds on mpv.io. The closest thing to an official build is lachs0r's and there's no newer build since October.
Shinchiro's builds use GCC <9, since 9 seems to have broken gpu-api=d3d11. Since this suite does not build GCC, we can't really go back to <9 anyway. Either use shinchiro's meanwhile, use gpu-api=vulkan or opengl, or propose a PR for the suite to build our own GCC as an option.

**Comment by wiiaboo** [comment]
The meson patch issue should be fixed in 8b73621.
